lib/api/controllers/authController.js

```diff
@@ -26,6 +26,9 @@ const
   _ = require('lodash'),
   User = require('../core/models/security/user'),
   formatProcessing = require('../core/auth/formatProcessing'),
+  {
+    IncomingMessage
+  } = require('http'),
```

Comment:
Nitpicking: 
Why don't use `IncomingMessage = require('http').IncomingMessage` ? I'm surprised that our linter allow that syntax for single import.

Reply:
It's the ES6 import (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)

Reply:
Nope. It's a [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)
Anyway, @Aschen is right, this is not useful to use that syntax when there is only 1 item to extract (and it's a nitpicking issue)

Reply:
Nodejs tends to promote destructured imports even for single inclusions.
cf https://nodejs.org/dist/latest-v8.x/docs/api/url.html#url_constructor_new_url_input_base for instance of node.js unit tests on github.

Reply:
I prefer this syntax too, even for a single import.  
I thought our linter won't allow this because I remember that I had to fix an import like that recently. Maybe it was before Node8 support